packages/peregrine/lib/talons/CheckoutPage/ShippingInformation/AddressForm/__tests__/__snapshots__/useGuestForm.spec.js.snap

```diff
@@ -22,6 +22,29 @@ Object {
 }
 `;
 
+exports[`handle submit fires mutation with street2 as null 1`] = `
```

Comment:
Is this still accurate? I don't see `street2` in this snapshot. I do see that the `street` array just has 1 variable but not a null for the second one.

Reply:
Yeah, in that case, `street2` is more the second value of the array. Without my correction, it would have been `['test', null]` which results in an error.
I can rename it like this: 'handle submit fires mutation with street[1] as null'.

Reply:
Wonderful, a rename would be perfect.